simple-loopback.json

```diff
@@ -54,6 +54,13 @@
       {
         "name": "local_service",
         "features": "http2",
+        "circuit_breakers": {
+          "default": {
+            "max_connections": 98304,
```

Comment:
You should just max these out. This won't cause Envoy to create all these connections.